File: 1480_RunningSumOf1dArray.js
// Given an array nums. 
// We define a running sum of an array as runningSum[i] = sum(nums[0]…nums[i]).
// Return the running sum of nums.

var runningSum = function(nums) {
    let sum = 0;
    let output = [];
    nums.forEach(n => {
        sum += n;
        output.push(sum);
    });
    return output;
};

File: Python/1480_RunningSumOf1dArray.py
class Solution(object):
    def runningSum(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        sum = 0
        output = []
        for n in nums:
            sum += n
            output.append(sum)
        return output

test = Solution()
print(test.runningSum([1,1,1,1,1]))
print(test.runningSum([1,2,3,4,5]))